code/game/objects/items/gamemode/survivor.dm

```diff
@@ -0,0 +1,340 @@
+/obj/item/laptop/rescue
+    name = "Communications laptop"
+    desc = "A device used for augmenting communication. Someone seems to have left it logged in with open communications to a nearby ship. Set this up next to a rescue beacon."
+    icon = 'icons/obj/machines/laptop_beacon.dmi'
+    icon_state = "tandy0"
+    luminosity = 4
+    resistance_flags = UNACIDABLE|INDESTRUCTIBLE
+    w_class = WEIGHT_CLASS_BULKY
+    var/activation_time = 5 SECONDS
+
+/obj/item/laptop/rescue/Destroy()
```

Comment:
This does nothing

Reply:
It did do something but you're right it should be removed.